perun-db/postgres.sql

```diff
@@ -762,6 +762,7 @@ create table groups_members (
 create table groups_resources (
 	group_id integer not null,     --identifier of group (groups.id)
 	resource_id integer not null,  --identifier of resource (resources.id)
+	status integer not null default 0,
```

Comment:
Did you think about creating the enum also in DB? I think it might be a better option since you don't need to do some custom mapping and you can be always sure that the DB will contain only the specified enum values, not something else.

Reply:
Also, I'm not sure if we want the `ACTIVE` status to be the default value. I think it would make sense to use the `PROCESSING` as a default value. What do you think?

Reply:
You are right. I created enum in DB and set the default value as PROCESSING.